db/controllers/articlesbyid.controller.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticleById } = require("../../db/models/articlebyid.model");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (!article_id || isNaN(Number(article_id))) {
```

Comment:
we can assume there will always be an article_id if we get into this endpoint so we don't need to check if !article_id - if we assert this we wouldn't be able to access an article_id of 0 if the ids were zero indexed